### PR TITLE
Provide naming policy on frontend

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/frontend/src/renderer/components/CreateJoinCommunity/CreateCommunity/CreateCommunity.test.tsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    expect(textInput).not.toBeNull()
+    userEvent.type(textInput, communityName)
+    const submitButton = result.queryByRole('button')
+    expect(submitButton).not.toBeNull()
+    expect(submitButton).toBeEnabled()
+    userEvent.click(submitButton)
+    await waitFor(() => expect(handleCommunityAction).toBeCalledWith(communityName))
+  })
+
+  it.each([
```
Comment (holmesworcester) [inline packages/frontend/src/renderer/components/CreateJoinCommunity/CreateCommunity/CreateCommunity.test.tsx:158]: Should this set of tests be copy pasted or a single function somewhere?

File: packages/frontend/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+      </>,
+      store
+    )
+
+    const createUsernameTitle = screen.getByText('Register a username')
+    expect(createUsernameTitle).toBeVisible()
+  })
+
+  // TODO: move this scenario to rtl-tests and provide socket (nectar sagas) to prepareStore method
+  it.skip('user rejoins to remembered community with user certificate', async () => {
```
Comment (holmesworcester) [inline packages/frontend/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx:273]: Should we do this now?

File: packages/frontend/src/utils/functions/naming.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const trimHyphen = (input: string): string => {
```
Comment (holmesworcester) [inline packages/frontend/src/utils/functions/naming.ts:1]: Shouldn't the tests be here, or is it better for them to be at a higher level?

**Review by holmesworcester — CHANGES_REQUESTED** [review]

**Review by vinkabuki — APPROVED** [review]

File: packages/frontend/src/renderer/components/CreateJoinCommunity/CreateCommunity/CreateCommunity.test.tsx (hunk 1 above)
Comment (siepra) [reply packages/frontend/src/renderer/components/CreateJoinCommunity/CreateCommunity/CreateCommunity.test.tsx:158]: I don't get it. Could you clarify your expectations?

File: packages/frontend/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx (hunk 2 above)
Comment (siepra) [reply packages/frontend/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx:273]: That's not related to this task at all. I've already told Damian that those test should be changed

**Pull request merged**